semeio/jobs/rft/gendata_rft.py

```diff
@@ -182,6 +182,8 @@ def run(
                 well=well, time=time
```

Comment:
How about just adding it as another entry here? 
`.assign(well=well, time=time, report_step=report_step)`

Reply:
yup! That is neater.